nano/node/bootstrap/bootstrap.cpp

```diff
@@ -1298,9 +1309,19 @@ nano::bootstrap_initiator::~bootstrap_initiator ()
 	stop ();
 }
 
-void nano::bootstrap_initiator::bootstrap ()
+void nano::bootstrap_initiator::bootstrap (bool force)
 {
 	nano::unique_lock<std::mutex> lock (mutex);
+	if (force)
+	{
+		if (attempt != nullptr)
```

Comment:
minor, but could probably combine the `if (attempt != nullptr)` statement into the one above and have `if (force && attempt != nullptr)` to save a scope & indentation.

Reply:
Sure